modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,157 @@
+            100000: 'Hauptfeuerwehrmann / Hauptfeuerwehrfrau',
+            1000000: 'Stv. Gruppenführer(in)',
+            5000000: 'Gruppenführer(in)',
+            10000000: 'Stv. Zugführer(in)',
+            20000000: 'Zugführer(in)',
+            50000000: 'Stv. Wehrführer(in)',
+            100000000: 'Wehrführer(in)',
+            200000000: 'Stv. Kreisbrandmeister(in)',
+            500000000: 'Kreisbrandmeister(in)',
+            1000000000: 'Stv. Landesbrandmeister(in)',
+            2000000000: 'Landesbrandmeister(in)',
+            5000000000: 'Ehrenmitglied'
+        },
+        texts:{
+            dropdownName: 'Creditserweiterung',
+            creditsOverview: 'Credits-Übersicht',
+            coinsProtokoll: 'Coinsprotokoll',
+            earnedCredits: 'Gesamtcredits',
+            creditsToNextRank: 'Credits zum nächsten Dienstgrad',
+            updateMessage: 'Werte aktualisieren',
+            allianceFunds: 'Verbandskasse',
+            noFurtherRank: 'Keine Höheren Ränge'
+        }
+    };
+    I18n.translations.en.lssm.creditserweiterung = {
+        earnedCreditsRegEx: /(.*)([0-9],)? earned Credits.*/i,
+        ranks: {
+            0: 'Probie',
+            200: 'Firefighter',
+            10000: 'Senior Firefighter',
+            100000: 'Fire Apparatus Operator',
+            1000000: 'Lieutenant',
+            5000000: 'Captain',
+            20000000: 'Staff Captain',
+            50000000: 'Battalion Chief',
+            1000000000: 'Division Chief',
+            2000000000: 'Deputy Chief',
+            5000000000: 'Fire Chief'
+        },
+        texts:{
+            reloadMessage: 'Please open your profile<br>and reload the website!',
+            dropdownName: 'Creditsextension',
+            creditsOverview: 'Overview',
+            coinsProtokoll: 'Coins List',
+            earnedCredits: 'Totally earned Credits',
+            creditsToNextRank: 'Credits to next rank',
+            updateMessage: 'Values are updated<br>all 5 minutes.',
+            allianceFunds: 'Alliance Funds',
+            noFurtherRank: 'There are no further ranks'
+        }
+    };
+    I18n.translations.nl.lssm.creditserweiterung = {
+        earnedCreditsRegEx: /(.*)([0-9]\.)? verdiende Credits.*/i,
+        ranks: {
+            0: 'Aspirant',
+            200: 'Brandwacht (Manschap A)',
+            10000: 'Hoofdbrandwacht (Manschap B)',
+            100000: 'Brandmeester',
+            1000000: 'Hoofdbrandmeester',
+            5000000: 'Commandeur',
+            50000000: 'Adjunct-Hoofdcommandeur',
+            500000000: 'Hoofdcommandeur',
+            5000000000: 'Erelid'
+        },
+        texts:{
+            reloadMessage: 'Open uw profiel<br>open en de website herladen!',
+            dropdownName: 'Kredietexpansie',
+            creditsOverview: 'Overzicht statistiek',
+            coinsProtokoll: 'Coin protocol',
+            earnedCredits: 'Totale studiepunten',
```

Comment:
Totaal aantal credits   (remove the "Totale studiepunten", and replace with this)